machine/nxp/nxp_ls1043ardb/kernel/config

```diff
@@ -0,0 +1,2960 @@
+#
```

Comment:
this is a pretty big kernel config for a machine.

the machine specific kernel config is supposed to be a small-ish diff from the CPU architecture kernel config: `build-config/conf/kernel/4.1.34/linux.armv8a.config`